templates/dashboard/support.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'default.html' %}
+
+{% set active_page = None %}
+
+{% block title %}
+    Support
+{% endblock %}
+
+{% block head %}
+    <style>
+        .card-title {
+            font-size: 22px;
+            font-weight: 600;
+            margin-bottom: 3px;
+        }
+    </style>
+
+    <script src="{{ url_for('static', filename='node_modules/vue/dist/vue.min.js') }}"></script>
+    <script>
+        $( document ).ready(function() {
+            var app = new Vue({
+                el: '#aliasGroup',
+                data: {
+                    ticketEmail: document.querySelector("input[name=ticketEmail]").value
```

Comment:
we can use Jinja2 variable here, for example `ticketEmail: "{{ current_user.email }}"`

Reply:
Ok..